packages/atomic/src/components/common/facets/numeric-facet/formatter.ts

```diff
@@ -0,0 +1,67 @@
+import {i18n} from 'i18next';
+import {getFieldValueCaption} from '../../../../utils/field-utils';
+import {NumberFormatter} from '../../formats/format-common';
+
+export interface FacetValueRange {
+  endInclusive: boolean;
+  start: number;
+  end: number;
+  label?: string;
+  state: 'selected' | 'idle' | 'excluded';
+}
+
+export interface FormatFacetValueRange {
+  field: string;
+  facetValue: FacetValueRange & {numberOfResults: number};
+  manualRanges: FacetValueRange[];
+  i18n: i18n;
+  logger: Pick<Console, 'error'>;
+  formatter: NumberFormatter;
+}
+
+export const formatHumanReadable = ({
+  manualRanges,
+  field,
+  i18n,
+  facetValue,
+  logger,
+  formatter,
+}: FormatFacetValueRange) => {
+  const manualRangeLabel = manualRanges.find((range) =>
+    areRangesEqual(range, facetValue)
+  )?.label;
+  return manualRangeLabel
+    ? getFieldValueCaption(field, manualRangeLabel, i18n)
+    : i18n.t('to', {
+        start: formatNumberLocalized(facetValue.start, i18n, logger, formatter),
+        end: formatNumberLocalized(facetValue.end, i18n, logger, formatter),
```

Comment:
When the the range is not endInclusive, should we subtract 1 from its end?

Reply:
Good point, that means this bug was there for a long time :D